src/citrine/resources/design_space.py

```diff
@@ -64,7 +64,11 @@ def update(self, model: DesignSpace) -> DesignSpace:
         self._validate_write_request(model)
         return AbstractModuleCollection.update(self, model)
 
-    def create_default(self, *, predictor_id: UUID) -> DesignSpace:
+    def create_default(self, *,
```

Comment:
```suggestion
    def create_default(self, 
                       *,
```
PEP has all args including the * on a new line.